### ISSUE TITLE
Features V1

### ISSUE THREAD

**Issue by d-gubert** [RocketChat/Apps.Atlassian.Jira#2]
Main features implemented by the official Jira Cloud app for Slack:

### Events
* [ ] Issue created
* [ ] Assignee updated
* [ ] Someone leaves a comment
* [ ] Issue transitioned (change status)

### Slash commands
* [ ] `/jira connect` - Instructions for installing the app in the Jira instance, including the url for the manifest
* [ ] `/jira DEMO-1234` - Get details about an issue
* [ ] `/jira help` - Show commands

**Comment by d-gubert** [comment]
When a comment has text modifications on Jira, Rocket.Chat is not showing it properly on the message attachment

_Jira_
![image](https://user-images.githubusercontent.com/1810309/47573239-33b66e80-d913-11e8-8fbe-244c536f7bac.png)

_Rocket.Chat_
![image](https://user-images.githubusercontent.com/1810309/47573301-5ea0c280-d913-11e8-885e-7520301067d8.png)

But Slack does adapt quite well to that

_Slack_
![image](https://user-images.githubusercontent.com/1810309/47573404-9ad42300-d913-11e8-892c-56a057872b16.png)

Do we deal with this in this first version or create an issue to work on it later?

@rodrigok what do you think?

**Comment by rodrigok** [comment]
@d-gubert uhmmmm, I think we can try to replace the formatting by our own, but as I can see that message part is not processing format since * should be formatted as bold.

**Comment by d-gubert** [comment]
Yeah, it looks like attachment descriptions are not processed like a message text. Or maybe I'm missing some configuration on the attachment?

**Comment by rodrigok** [comment]
@d-gubert nice, how does this renders now?

**Comment by d-gubert** [comment]
![image](https://user-images.githubusercontent.com/1810309/47667025-bf363680-db83-11e8-8b2c-c4f3baa2aca8.png)

Properly formats bold and italic, but not the others (obviously). I think we should "translate" the formatting we support in Rocket.Chat, such as strike through, and strip the rest off
